portfolio/src/main/webapp/index.html

```diff
@@ -1,48 +1,56 @@
 <!DOCTYPE html>
 <html>
-
+ 
 <head>
-	<meta charset="UTF-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-	<title>My Portfolio</title>
-	<!-- CDN link to locate Bootstrap -->
-
-	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
-		integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
-	<link rel="stylesheet" href="style.css">
-
-	<script src="script.js"></script>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <title>My Portfolio</title>
+    <!-- CDN link to locate Bootstrap -->
+ 
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
+        integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <link rel="stylesheet" href="style.css">
+ 
+    <script src="script.js"></script>
 </head>
-
+ 
 <body>
-	<nav class="navbar fixed-top navbar-dark bg-dark">
-		<div class="container-fluid">
-			<div class="navbar-header">
-				<span class="navbar-brand mb-0 h1">Prashant Neupane</span>
-			</div>
-
-			<ul class="navbar-nav ml-auto">
-				<li class="nav-item">
-					<a class="nav-link" href="#">HOME</a>
-				</li>
-
-				<li class="nav-item">
-					<a class="nav-link" href="#AboutMe">ABOUT </a>
-				</li>
-
-				<li class="nav-item">
-					<a class="nav-link" href="#Connect">CONNECT</a>
-				</li>
-			</ul>
-		</div>
-	</nav>
-
-		<h1>My Portfolio</h1>
-		<p>This is my portfolio.</p>
-		<p>Click here to get a random message:</p>
-		<button onclick="addRandomGreeting()">Hello</button>
-
-		<div id="greeting-container"></div>
+    <nav class="navbar fixed-top navbar-dark bg-dark">
+        <div class="container-fluid">
+            <div class="navbar-header">
+                <span class="navbar-brand mb-0 h1">Prashant Neupane</span>
+            </div>
+ 
+            <ul class="navbar-nav ml-auto">
+                <li class="nav-item">
+                    <a class="nav-link" href="#">HOME</a>
+                </li>
+ 
+                <li class="nav-item">
+                    <a class="nav-link" href="#AboutMe">ABOUT </a>
+                </li>
+ 
+                <li class="nav-item">
+                    <a class="nav-link" href="#Connect">CONNECT</a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+ 
+    <div id="parallax">
+        <div class="quotes">
+            <span class="border">
+                I think therefore I am - Prashant Neupane
```

Comment:
This is definitely not my quote. I forgot to take it out. This quote is by Rene Descartes.